CONTRIBUTING.md

```diff
@@ -30,6 +30,8 @@ Please select the `master` branch as the destination for your Pull Request so yo
 
 Please squash your changes into a single commit using a command like `git rebase -i upstream/master`.
 
+To test C++ changes, you can compile the module using `npm run compile`.
```

Comment:
Sorry, my mistake. Judging by the command line history `npm i && npm t` will do both.